### DISCUSSION TITLE
Register a Bearware.dk account

### DISCUSSION THREAD

**Discussion by rexya2017** [BearWare/TeamTalk5#2005] [Q&A]
### Description

Some people have difficulty registering on bearware login. This does not work as it should, once the registration is complete, the Bearware does not send a confirmation email so the user cannot set his password. Of course, bearware login cannot be done in TeamTalk 5, on all platforms.

### Application

- [X] qtTeamTalk
- [X] TeamTalkAndroid
- [X] iTeamTalk
- [X] TeamTalkClassic
- [X] TeamTalkServer

### Platform

- [X] Windows
- [X] macOS
- [X] Android
- [X] iOS
- [X] Linux

### Expected behavior

After registration is done, the system should send a confirmation email, or email to set the password.

### Actual behavior

The confirmation Email is not sent by the system after the user registers.

### Steps to reproduce problem

go to bearware.dk and click on the register, or open this URL: https://bearware.dk/wordpress/wp-login.php?action=register
Fill in all the required fields such as username, email, first name, last name, display name.
then click on the Register button.
Next, open your email that has been previously registered above, then confirm by opening the link / URL that has been sent by the system, to confirm your account.

**Comment by bear101** [comment]
Do you have a username or First Name of someone who didn't manage to register?

**Comment by rexya2017** [comment]
This is one of the usernames I know, and out there are still some people who can't sign up.
username: AraPearl
firstname: Ridha
The resulting problem remains the same, the confirmation email is not sent to the username above.

**Reply by bear101** [reply]
I've sent a password reset link again to this username. Please check and see if it arrives. Otherwise I can set a password manually.

**Reply by rkp9090** [reply]
Please Also Send Me Email For Username: zul.
Because I also experienced the same problem.

**Reply by bear101** [reply]
Please send from the email associated with username zul.

**Comment by rexya2017** [comment]
Thanks, I'll tell the guy. But my question is, Will the confirmation email
system not be fixed? Wouldn't it be difficult for you if you had to set a
password by sending a password reset URL manually?

Pada tanggal Kam, 7 Sep 2023 pukul 03.33 Bjørn Damstedt Rasmussen <
***@***.***> menulis:

**Comment by raf-li** [comment]
I, as a TeamTalk user for more than 5 years, am actually a bit disappointed with the breakdown of the system that automates sending links to confirm or reset passwords in this bearware. I tried a few days ago to reset the password and, no email came even till this day, I tried to do it again but still, though I tried to create a new account with a different email, it was the same, so I'm sure that this problem didn't come from a typo that I made, but the system didn't send the links at all.
And, you should think about fixing your broken system, instead of sending mail manually, or even setting the person's password manually, it will take more time if more people have the same problem.

**Comment by rexya2017** [comment]
This is what I meant, instead of sending a password reset manually, why not
fix it from the system side only.

Pada tanggal Kam, 7 Sep 2023 pukul 07.49 reffi2269 ***@***.***>
menulis:

**Comment by rkp9090** [comment]
Ok, I will.

Pada tanggal Sen, 25 Sep 2023 01.10, Bjørn Damstedt Rasmussen <
***@***.***> menulis:

**Comment by computertech1012** [comment]
Hi,
There are still problems sometimes with this. And also, as of the time of this writing, the stats bot is not logging into any of the unoficial servers.
@bear101 can you please add a more reliable way for your servers to not always have down time or problems, because even now user can't publish their server in the list, and at the time of this message the stats bot is down once again. So not only is a bearware registration sometimes impossible, but the stats bot is so buggy and unreliable.
Thanks for reading!
